### PR TITLE
Clarify licensing of trademarks in FAQ

### PR DESCRIPTION
**Pull request by thisandagain**
### Resolves

- Resolves some common issues that are often raised in our helpdesk system about the licensing of the Scratch logo, cat, and related trademarks.

### Changes

- More clearly states that the Scratch logo, cat, and characters (Gobo, Pico, Tera, Nano) are trademarks and removes some misleading wording in the FAQ that was sending mixed messages.

/cc @christanbalch @dietbacon

### CODE REVIEW

File: src/views/faq/l10n.json [hunk 1]
```diff
@@ -75,14 +75,14 @@
...
     "faq.presentScratchTitle":"Can I present Scratch at a conference?",
     "faq.presentScratchBody":"Please feel free to make presentations about Scratch to educators or other groups. We grant our permission to make presentations.",
     "faq.supportMaterialTitle":"May I use / remix Scratch support materials, sprites, images, sounds or sample projects I’ve found on the website?",
-    "faq.supportMaterialBody":"Yes - Scratch support materials made available on the Scratch website by the Scratch Team are available under the <a href=\"http://creativecommons.org/licenses/by-sa/2.0/deed.en\">Creative Commons Attribution-ShareAlike license</a>, with the exception of the Scratch Logo, Scratch Cat, and Gobo (Scratch trademarks).",
+    "faq.supportMaterialBody":"Yes - Scratch support materials made available on the Scratch website by the Scratch Team are available under the <a href=\"http://creativecommons.org/licenses/by-sa/2.0/deed.en\">Creative Commons Attribution-ShareAlike license</a>, with the exception of the Scratch Logo, Scratch Cat, Gobo, Pico, Nano, and Tera which are Scratch tradmarks.",
```
Comment (chrisgarrity) [inline src/views/faq/l10n.json]: typo in 'trademarks'

Comment (thisandagain) [reply src/views/faq/l10n.json]: 😫

Comment (thisandagain) [reply src/views/faq/l10n.json]: Fixed

**Comment by thisandagain** [comment]
Note to self: squash during merge

**Pull request merged**